.github/workflows/tests.yml

```diff
@@ -28,26 +47,15 @@ jobs:
           sudo lxd init --auto
           sudo snap install snapcraft --classic
           snap list
+          sudo apt update
+          sudo apt -y install bats
+          sudo modprobe openvswitch
 
       - name: Build snaps
-        run: snapcraft
+        run: make microovn.snap
```

Comment:
Technically, yes, but there was previously a separate step for building the snap, and I think we should keep it.

Having it as a separate step have multiple benefits:
1) In the job output it gets its own section that can be collapsed separately from the actual test output.
2) Down the line, we may have multiple jobs that rely on the built artifact and we may want to share it across those jobs instead of building multiple times.